src/xml_errors.cc

```diff
@@ -5,28 +5,30 @@
 
 using namespace v8;
 
-void set_string_field(Local<Object> obj, const char *name, const char *value) {
+void set_string_field(Local<Object> obj, const char *name, const char *value)
+{
   Nan::HandleScope scope;
-  if (!value) {
+  if (!value)
+  {
     return;
   }
   Nan::Set(obj, Nan::New<String>(name).ToLocalChecked(),
            Nan::New<String>(value, strlen(value)).ToLocalChecked());
 }
 
-void set_numeric_field(Local<Object> obj, const char *name, const int value) {
+void set_numeric_field(Local<Object> obj, const char *name, const int value)
+{
   Nan::HandleScope scope;
   Nan::Set(obj, Nan::New<String>(name).ToLocalChecked(),
            Nan::New<Int32>(value));
 }
 
 // Code copied from libxmljs. Should be kept in sync. Or dynamically linked to libxml node extension.
-Local<Value> BuildSyntaxError(xmlError *error) {
+Local<Value> BuildSyntaxError(xmlError *error)
+{
   Nan::EscapableHandleScope scope;
 
-  Local<Value> err =
-      Exception::Error(Nan::New<String>(error->message).ToLocalChecked());
-  Local<Object> out = Local<Object>::Cast(err);
+  Local<Object> out = Nan::New<Object>();
```

Comment:
Why wouldn't we want to create a JavaScript Error object ?
IMHO, it is useful since it indicates that this is an error/exception that could also be thrown, or logged with a proper stack trace.